### ISSUE TITLE
Preserve unavailable choices

### ISSUE THREAD

**Issue by jace** [hasgeek/hasjob#125]
When editing a listing, if the existing job type, category or pay currency is not in the choices, add it. This required when editing a listing across boards.

**Comment by sindhus** [comment]
Wont allowing users to add their own job type/category and pay currency create duplicates?

**Comment by jace** [comment]
No, this for items in the database but currently unavailable (retired category or not available on this sub-board). If the job already links to it, editing the job will mean losing the category.
